CMakeLists.txt

```diff
@@ -27,21 +27,27 @@ enable_testing()
 set(CMAKE_CXX_STANDARD 17)
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
 
+# Suppress the warning related to the new policy on fetch content's timestamp
+if(CMAKE_VERSION VERSION_GREATER_EQUAL "3.24.0")
+      cmake_policy(SET CMP0135 NEW)
+endif()
+
 # Location of current CPU torch
 include(FetchContent)
 FetchContent_Declare(
-  torch
-  URL https://download.pytorch.org/libtorch/nightly/cpu/libtorch-shared-with-deps-latest.zip
+      torch
+      URL https://download.pytorch.org/libtorch/nightly/cpu/libtorch-shared-with-deps-latest.zip
 )
 
 # Select between system torch and the current (CPU) torch
-if (DEFINED LIBTORCH_DIR)
-      find_package(Torch REQUIRED HINTS "${LIBTORCH_DIR}/share/cmake") 
-else()
+if(NOT DEFINED LIBTORCH_DIR)
+      Set(FETCHCONTENT_QUIET FALSE)
```

Comment:
This prints out the download progress. It is mainly for preventing the awkward silence especially when you have a slow internet (like my home internet).